src/components/shared/SimpleSelect.tsx

```diff
@@ -12,28 +13,46 @@ const customStyles = {
     '&:hover': {
       borderColor: '#B8BBBF',
     },
+    minHeight: '100%',
     fontSize: '14px',
     overflow: 'hidden',
     outline: 'none',
+    padding: '0px',
   }),
-  placeholder: (provided: any) => ({
-    ...provided,
-    color: 'transit-black',
-    fontSize: 14,
+  menuList: (base: any) => ({
+    ...base,
+    // kill the white space on first and last option
```

Comment:
@KarolinaKopacz is this comment necessary?

Reply:
I can remove it in this place but just in one it need to stay to understand why padding - 0px is here :)

Reply:
ok